infra/terraform/test_fixtures/terraform-google-iam.tf

```diff
@@ -0,0 +1,80 @@
+locals {
+  // Roles required by CI service accounts to run the IAM
+  // integration tests.
+  iam_required_roles = [
+    "roles/resourcemanager.organizationAdmin",
+    "roles/compute.xpnAdmin",
+    "roles/storage.admin",
+    "roles/pubsub.admin",
+    "roles/cloudkms.admin",
+  ]
+}
+
+resource "google_project" "iam" {
+  provider = "google.phoogle"
+  name = "ci-iam"
+  project_id = "ci-iam"
+  folder_id = "${google_folder.phoogle_cloud_foundation_cicd.name}"
+  billing_account = "${module.variables.phoogle_billing_account}"
+}
+
+resource "google_project_services" "iam" {
+  provider = "google.phoogle"
+
+  project = "${google_project.iam.id}"
+  services = [
```

Comment:
The following services are missing:

Cloud Key Management Service
Cloud Pub/Sub
Google Cloud Storage JSON

Reply:
Addressed in a5a955f Add missing services for IAM Fixture